tests/e2e/hcp_cluster_test.go

```diff
@@ -32,6 +32,13 @@ var _ = Describe("HCP cluster testing",
 			var err error
 			clusterConfig, err = config.ParseClusterProfile()
 			Expect(err).ToNot(HaveOccurred())
+
+			By("Skip testing if the cluster is not a HCP cluster")
+			hostedCluster, err := clusterService.IsHostedCPCluster(clusterID)
+			Expect(err).ToNot(HaveOccurred())
+			if !hostedCluster {
+				SkipTest()
```

Comment:
I would rather a more explicit message than just `skip on profile`. The message is the results would not be explicit enough and hard to review if needed.
Or add `SkipNotHosted`, `SkipNotClassic`  methods

Reply:
Yes, I have thought of users `SkipHCPTest` and `SkipClassicTest` in the past. But it is strange that we have judged `if !hostedCluster`.  
```
			if !hostedCluster {
				SkipClassicTest()
```


Seems like this is better.
```
			if !hostedCluster {
				SkipNotHosted()

Reply:
yeah second option looks better to read